images/linux/scripts/installers/basic.sh

```diff
@@ -12,11 +14,4 @@ for package in $common_packages $cmd_packages; do
     apt-get install -y --no-install-recommends $package
 done
 
-# Run tests to determine that the software installed as expected
-echo "Testing to make sure that script performed as expected, and basic scenarios work"
-for cmd in $cmd_packages; do
-    if ! command -v $cmd; then
-        echo "$cmd was not installed"
-        exit 1
-    fi
-done
+invoke_tests "Cmd"
```

Comment:
what about tests for common_packages? Why don't we have it?

Reply:
if the package can not be installed on installation stage it will throw an error so tests are not needed

Reply:
ok, I see, thanks